image-registry/remove-registry-baremetal.md

```diff
@@ -0,0 +1,137 @@
+---
+title: remove-registry-baremetal
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@dmage"
+  - "@bparees"
+  - "@sdodson"
+  - "@rgolangh" # RHV
+  - "@russellb" # k8s-native infra
+approvers:
+  - "@bparees"
+  - "@smarterclayton"
+  - "@derekwaynecarr"
+creation-date: 2019-09-25
+last-updated: 2019-09-25
+status: provisional
+see-also: 
+replaces:
+superseded-by:
+---
+
+# Remove Registry on Baremetal
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+On `baremetal` platform clusters, the OpenShift image registry operator will bootstrap itself as `Removed`.
+This will allow `openshift-installer` to complete installations on this platform type.
+A separate process will be responsible for configuring storage and updating the image registry operator's state to `Managed`.
```

Comment:
I guess a CSI plugin for RHV can be used to provide storage for the registry? I need to port the flexplugin I wrote for 3.11 to CSI anyway... 
The 'process' you refer to, that will update the registry state, is this something I need to create for 'ovirt' platform or should it be a part of the registry operator?

Reply:
> I guess a CSI plugin for RHV can be used to provide storage for the registry? I need to port the flexplugin I wrote for 3.11 to CSI anyway...

I don't think that to develop a new storage back-end for the Image Registry would be part of this task but I would love to read any resource you could throw at me when it comes to CSI plugins.

> The 'process' you refer to, that will update the registry state, is this something I need to create for 'ovirt' platform or should it be a part of the registry operator?

I don't think it will be part of the Operator. This operation requires provisioning the storage first(probably through a manual user intervention) and later configuration on the Operator.

Reply:
@rgolangh can your flexplugin provide RWX volumes?

Reply:
As for the plugin to provide storage for the registry: this is out of scope for this ticket.